### ISSUE TITLE
Questions for reproducing the performance

### ISSUE THREAD

**Issue by youngbin-ro** [zhanjunlang/Span_OIE#3]
Hi,
I trained the model with the uploaded code, but couldn't reproduce the performance on the paper.
Is the settings in the code, such as random seed, hyper-parameters, and model architecture, the same as that shown in the paper?

I revised some settings like pos_dim and syntax_flag following the paper configurations.
Is there additional things for me to consider to train the model?
or could you release the trained model for future research?

Thank you.

**Comment by 131250208** [comment]
> Hi,
> I trained the model with the uploaded code, but couldn't reproduce the performance on the paper.
> Is the settings in the code, such as random seed, hyper-parameters, and model architecture, the same as that shown in the paper?
> 
> I revised some settings like pos_dim and syntax_flag following the paper configurations.
> Is there additional things for me to consider to train the model?
> or could you release the trained model for future research?
> 
> Thank you.

Could you share your results?
